packages/nitro-protocol/contracts/ForceMove.sol

```diff
@@ -814,16 +814,6 @@ contract ForceMove is IForceMove {
             );
     }
 
-    /**
-     * @notice Computes the hash of a given outcome.
-     * @dev Computes the hash of a given outcome.
-     * @param outcome An outcome
-     * @return The outcomeHash
-     */
-    function _hashOutcome(bytes memory outcome) internal pure returns (bytes32) {
-        return keccak256(abi.encode(outcome));
-    }
-
```

Comment:
We were advised to reuse this function instead of sometimes directly doing `keccak256(abi.encode(...))`. However: I spotted that we don't actually need to `abi.encode` here at all. Removing that makes `_hashOutcome` an alias for `keccak256` and therefore redundant.